packages/fx-core/tests/plugins/resource/bot/unit/index.test.ts

```diff
@@ -166,16 +165,17 @@ describe("Teams Bot Resource Plugin", () => {
       const fakeCreds = testUtils.generateFakeTokenCredentialsBase();
 
       let item: any = { registrationState: "Unregistered" };
-      const namespace = ["ut"];
       const fakeRPClient: any = {
+        // eslint-disable-line no-unused-vars
```

Comment:
If we don't use rpClient any more, does it mean we don't need to mock it and remove unused code?

Reply:
nod. but still want to check I will not call some API anymore, so I used 
`sinon.assert.notCalled`
check that behavior.